custom_components/tahoma/executor.py

```diff
@@ -4,13 +4,23 @@
 from typing import Any
 from urllib.parse import urlparse
 
-from pyoverkiz.enums.command import OverkizCommand
+from pyoverkiz.enums import OverkizCommand, Protocol
 from pyoverkiz.models import Command, Device
 from pyoverkiz.types import StateType as OverkizStateType
 
 from .const import LOGGER
 from .coordinator import OverkizDataUpdateCoordinator
 
+# Commands that don't support setting
+# the delay to another value
+COMMANDS_WITHOUT_DELAY = [
+    OverkizCommand.IDENTIFY,
+    "test",
```

Comment:
I guess you will use the new pymfy version to replace this string?

Reply:
Yes! :D

Reply:
Done